src/crypto/public_key/elliptic/ecdh_x.js

```diff
@@ -3,7 +3,8 @@
  * @module crypto/public_key/elliptic/ecdh
  */
 
-import nacl from '@openpgp/tweetnacl/nacl-fast-light';
+import X25519 from '@openpgp/tweetnacl/nacl-fast-light';
+import { x448 as X448 } from '@openpgp/noble-curves/ed448';
```

Comment:
This is extremely nitpicky but since these are not classes, I'd import them as lowercase